_episodes/16-parallel.md

```diff
@@ -74,13 +74,15 @@ def inside_circle(total_count):
 {: .language-python}
 
 > ## Random Number Generation
-> 
-> For simplicity, random numbers are generated using regular functions, if
-> a high quality random numbers are needed in parallel applications, one should
-> use libraries such as [SPRNG](http://sprng.org/) and 
-> [Kmath-random](https://www.r-ccs.riken.jp/labs/lpnctrt/projects/kmath-random/).
-> * Discuss why generating high quality random numbers in parallel might be 
-> difficult.
+> In the preceding code, random numbers are conveniently generated using the 
```

Comment:
Better phrasing than we had before! For the benefit of Instructors, we should include a Solution sub-box with answers to the questions posed here. 

```markdown
> > ## Solution
> >
> > * Computers are deterministic
> > * Yes
> > * This is not a suitable discussion question, as it is addressed to a self-taught learner rather than the classroom (virtual or physical).
> > {: .solution}
```

Reply:
Thanks. Suggestions implemented.